templates/scenarios/js/dashboard-tab/README.md

```diff
@@ -34,7 +34,7 @@ This app also supported teams different themes, including dark theme and high co
 
 ## Prerequisites
 
-- [Node.js](https://nodejs.org/), fully tested on NodeJS 14, 16, 18
+- [Node.js](https://nodejs.org/), supported versions: 16, 18
```

Comment:
Is 14 really not supported or just not fully tested/recommended?
OK to change later but prefer to confirm the wording. Since this may confuse user "*I can only use node 16 and 18*" (BTW, 20 is just available...)

Reply:
> Is 14 really not supported or just not fully tested/recommended? OK to change later but prefer to confirm the wording. Since this may confuse user "_I can only use node 16 and 18_" (BTW, 20 is just available...)

It means not fully tested. But we prefered the new users to use LTS node.js versions. 
The wording is provided by PM in the node 18 support feature. We can confirm with PM again.

Reply:
Comfirmed with PM, the supported version has been used in the docs site. Here we can keep same with docs site, the supported versions is enough.